integration-libs/cds/src/profiletag/services/profile-tag-push-events.service.ts

```diff
@@ -350,6 +355,22 @@ export class ProfileTagPushEventsService {
     );
   }
 
+  protected cartChangedEvent(): Observable<ProfileTagPushEvent> {
+    return merge(
+      this.eventService.get(CartAddEntrySuccessEvent),
+      this.eventService.get(CartUpdateEntrySuccessEvent),
+      this.eventService.get(CartRemoveEntrySuccessEvent)
+    ).pipe(
+      switchMapTo(this.activeCartFacade.takeActive()),
```

Comment:
Let me know if this is working fine for you now~ 

If so, we'll wait for https://github.com/SAP/spartacus/pull/15716 to be merged

Finally, once this is merged, make sure to backport it to `release/4.3.x` to be released as a patch. However, do not forget to handle the constructor overload. An example of handling constructor deprecation can be found [here](https://github.com/SAP/spartacus/blob/release/4.3.x/projects/storefrontlib/cms-components/cart/add-to-cart/add-to-cart.component.ts#L58-L84)

Reply:
don't forget to run yarn prettier:fix in root to prettify your files if you don't have prettier enabled in your ide